dds/DCPS/Recorder.h

```diff
@@ -16,7 +16,7 @@
 
 #include <dds/DdsDcpsInfrastructureC.h>
 
-#include "XTypes/DynamicDataImpl.h"
+#include "XTypes/DynamicDataXcdrReadImpl.h"
```

Comment:
```suggestion
#include "<dds/DdsDynamicDataC.h>
```
Only the interface types are needed in the header